vendor/knative.dev/pkg/injection/sharedmain/main.go

```diff
@@ -120,12 +119,15 @@ var (
 // by name.
 func MainNamed(ctx context.Context, component string, ctors ...injection.NamedControllerConstructor) {
 
-	disabledControllers := pflag.StringSlice("disable-controllers", []string{}, "Comma-separated list of disabled controllers.")
+	var (
+		disabledControllers string
+	)
+	flag.StringVar(&disabledControllers, "disable-controllers", "", "Comma-separated list of disabled controllers.")
```

Comment:
```suggestion
	disabledControllers := flag.String("disable-controllers", "", "Comma-separated list of disabled controllers.")
```

Reply:
I think most eventing does use the `flag.StringVar` - that's why :man_shrugging:

Reply:
/lgtm